### DISCUSSION TITLE
Trouble deploying react-app on Vercel. Build Errors

### DISCUSSION THREAD

**Discussion by owenosa** [vercel/vercel#5506] [Help]
I'm new to deploying apps, and decided to use Vercel. I keep getting the same build error as shown below. I'm not sure what direction to go in to solve this issue. If anyone can help please let me know. 

https://github.com/owenosa/cryptodash

<img width="540" alt="Screen Shot 2020-12-02 at 10 34 17 AM" src="https://user-images.githubusercontent.com/57696998/100901973-0b05e900-348a-11eb-9ed8-d56a335fb35b.png">

**Comment by wiiiimm** [comment]
@owenosa I just deployed your repo using vercel-cli (`vercel link` and `vercel`) and it was deployed fine without an issue.

![image](https://user-images.githubusercontent.com/179761/100962608-9ac99700-355f-11eb-8bd1-616e1bd9f3b7.png)

It also seems that your latest commit was deployed fine.

![image](https://user-images.githubusercontent.com/179761/100962655-b92f9280-355f-11eb-926d-11c9ea29707d.png)

I guess you have fixed your issue already 👍 .
